packages/launchpad/src/migration/MajorVersionWelcome.vue

```diff
@@ -27,81 +27,69 @@
-              </p>
-              <p>
-                If your existing tests relied on a previous test to run successfully, you might need to make some modifications to your test suite. See the
-
-                <ExternalLink href="https://on.cypress.io/migration-guide#Migrating-to-Cypress-12-0">
-                  Migration Guide
-                </ExternalLink>
-
-                for more details on what you can expect.
+                Test Replay is available in all Cypress Cloud plans. To start using Test Replay, simply record a run to Cypress Cloud. Learn more in our <ExternalLink href="https://on.cypress.io/test-replay?utm_source=splash-page&utm_campaign=v13">
+                  documentation
+                </ExternalLink>.
               </p>
 
               <h2 class="font-bold mt-[24px] mb-[16px] text-[18px] text-jade-1000">
-                We Now Store The Browser Context, So You Don’t Have To
+                Changes to video defaults
               </h2>
 
               <p>
-                The <ExternalLink href="https://on.cypress.io/session">
-                  <code>cy.session()</code>
-                </ExternalLink> command complements test isolation by providing a way to save and share browser contexts between tests and specs in a single run on the same machine.
+                <code>video</code> and <code>videoCompression</code> are now set to <code>false</code> by default. Recording and compressing videos can be resource intensive, often impacting the duration of tests running in CI. Test Replay now serves as the primary replacement for debugging via video. We’ve changed these video options to be opt-in so you can configure recording video based on your needs.
               </p>
 
-              <h2 class="font-bold mt-[24px] mb-[16px] text-[18px] text-jade-1000">
-                Detaching Ourselves From Detached Dom Errors
-              </h2>
-
               <p>
-                We have made enhancements to how Cypress manages DOM element resolution to reduce the likelihood of hitting the dreaded detached DOM errors due to maintaining stale DOM references. We've updated our
-
-                <ExternalLink href="https://on.cypress.io/retry-ability">
-                  Retry-ability Guide
-                </ExternalLink>
-
-                with all the details if you'd like to learn more.
+                The <code>shouldUploadVideoOnPass</code> option is no longer available. This option was mostly used to skip video compression for unused videos which is now skipped by default. If you want to control which videos are saved locally or uploaded to Cypress Cloud, see our <ExternalLink href="https://on.cypress.io/migration-guide?utm_source=splash-page&utm_campaign=v13">
```

Comment:
What would you think of using `<InlineCodeFragment>` here instead of plain `code` elements. It has a little bit better visual separation from the surrounding text and looks more like what we do elsewhere.

Reply:
I can update the references and push it up!

Reply:
IMO I find it cleaner 
![](https://github.com/cypress-io/cypress/assets/3980464/f30c6bf6-69db-42a9-b343-914de592e299)